gitbutler-ui/src/lib/components/GithubIntegration.svelte

```diff
@@ -102,15 +100,15 @@
 		</svelte:fragment>
 		<Button {disabled} kind="filled" color="primary" on:click={gitHubStartOauth}>
 			{#if $user$?.github_access_token}
-				Reauthenticate
+				Reauthorize
```

Comment:
As discussed, we might want to allow the user to first remove any existing integration before offering to set it up again.

Reply:
Fixed that with the commit below